src/sockperf.cpp

```diff
@@ -88,11 +88,17 @@
 #include "aopt.h"
 #include <stdio.h>
 #include <sys/stat.h>
+#include <regex>
 
 #ifndef WIN32
 #include <dlfcn.h>
 #endif
 
+#if defined(__GNUC__) && (__GNUC__ < 4 || (__GNUC__ == 4 && __GNUC_MINOR__ < 9))
+#   include <regex.h>
+#   define NEED_REGEX_WORKAROUND
+#endif
```

Comment:
Please double check if `#if include <regex.h> #else #include <regex> #endif` is needed

Reply:
Yes, I checked that std::regex is not working with gcc 4.8.5 (default gcc version in RHEL 7.6). This is a well-known limitation, so I forced to add this condition.

Reply:
Let clarify I see that there is possibility to have both include `<regex.h>` and `<regex>` at the same time. Probably it is not good.

Reply:
make this includes mutually exclusive